gcp/database/main.tf

```diff
@@ -8,8 +8,14 @@ resource "google_sql_database_instance" "master" {
     ip_configuration {
       ipv4_enabled = true
 
-      authorized_networks {
-        value = var.network
+      dynamic "authorized_networks" {
+        for_each = var.cloud_sql_authorized_ips
+        iterator = allowed_ip
+
+        content {
+            name  = "allowed_ip.value.name"
```

Comment:
@cvalaas I don't think you need quotes around allowed_ip.value.name and ip_range

Reply:
rm'd